tests/unit/test_hardware.py

```diff
@@ -120,6 +120,13 @@ def test_parse_maximal_constraint() -> None:
         network:
             - type: eth
             - type: eth
+        system:
+            vendor: 0x413C
+            vendor-name: "~ Dell.*"
+            model: 79
+            model-name: "~ PowerEdge R750"
+            numa-nodes: "< 4"
+            lab-controller: "!~ lab-%.foo.bar.com"
```

Comment:
The `lab-controller` part should be dropped as it is now under `location`, right?

Reply:
Fixed in 5508cee5

Reply:
I'm pretty sure this one's addressed.